lib/checkDupHostingKeys.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var _ = require('lodash');
+var RSVP = require('rsvp');
+var utils = require('./utils');
+var chalk = require('chalk');
+var Config = require('./config');
+var FirebaseError = require('./error');
+var logger = require('./logger');
+
+module.exports = function(options) {
+  return new RSVP.Promise(function(resolve, reject) {
+    if (!options.config) {
+      return reject(new FirebaseError('Missing firebase.json.'));
```

Comment:
I think there's already an error message for this somewhere -- either use the same message or re-configure the order of "befores" to make this one after that error would throw.